exam/src/main/java/in/testpress/exam/util/GetEmailOrUsernameUtil.java

```diff
@@ -0,0 +1,41 @@
+package in.testpress.exam.util;
+
+import androidx.fragment.app.FragmentActivity;
+import in.testpress.core.TestpressCallback;
+import in.testpress.core.TestpressException;
+import in.testpress.core.TestpressUserDetails;
+import in.testpress.models.ProfileDetails;
+
+public class GetEmailOrUsernameUtil {
```

Comment:
This code needs cleanup.

1. Too many null checks. This means there is no guarantee in the code being used and hence implies lots of doubts.
2. The name overloading is confusing. getEmailOrUsername is present as part of class name as well as two methods name leading to ambiguity.